### PR TITLE
Feat/data upload

### PR DESCRIPTION
**Pull request by isaackps**
# In this PR:
- Update the Ui to the latest one created by Jerry
- When user download a sample data file, it will consist of their form key.

https://www.figma.com/file/vTVuTF2yuEQpvAlfDCYpOh/.CSV-.JSON-Schema?node-id=0%3A1

### CODE REVIEW

File: src/components/DynamicFormContainer/DataFileButton/DataFileButton.tsx [hunk 1]
```diff
@@ -58,7 +67,7 @@ export const DataFileButton: FunctionComponent<DataFileButton> = ({ onDataFile,
   };
 
   // eslint-disable-next-line @typescript-eslint/no-shadow
-  const validateDataFile = (schema: Schema, data: unknown): ValidateDataFile => {
+  const validateDataFile = (schema: AnySchema, data: unknown): ValidateDataFile => {
```
Comment (cavacado) [inline src/components/DynamicFormContainer/DataFileButton/DataFileButton.tsx:70]: this schema type here cannot use `JSONSchema`?

Comment (isaackps) [reply src/components/DynamicFormContainer/DataFileButton/DataFileButton.tsx:70]: ok will update to use JSONSchema

**Review by cavacado — COMMENTED** [review]
looks okay, make sure that the ci/cd passes then i'll approve this

**Review by cavacado — APPROVED** [review]
lgtm

**Pull request merged**